app/code/Magento/InventorySales/Model/IsProductSalableForRequestedQtyCondition/IsSalableWithReservationsCondition.php

```diff
@@ -89,8 +89,10 @@ public function execute(string $sku, int $stockId, float $requestedQty): Product
 
         $qtyWithReservation = $stockItemData[GetStockItemDataInterface::QUANTITY] +
             $this->getReservationsQuantity->execute($sku, $stockId);
-        $qtyLeftInStock = $qtyWithReservation - $stockItemConfiguration->getMinQty() - $requestedQty;
-        $isEnoughQty = (bool)$stockItemData[GetStockItemDataInterface::IS_SALABLE] && $qtyLeftInStock >= 0;
+        $qtyLeftInStock = $qtyWithReservation - $stockItemConfiguration->getMinQty();
+        $isInStock = bccomp((string)$qtyLeftInStock, (string)$requestedQty, 4) >= 0;
```

Comment:
How is that different from `$qtyLeftInStock - $requestedQty > 0.0001` on such scale?

Whatever epsilon will be, please put it as a constant in framework instead of hardcoding, feel free to use it for other occurrences introduced before.

Reply:
I've read all the comments prior to review. Proper eps should do the trick, it may be `1e-9` or `1e-17` though. `bccomp` is not needed on such scale.

> we dont have currently an const for count of decimals

That's exactly what I'm asking for - it MUST be created instead of hardcoding random small values.

Reply:
>How is that different from $qtyLeftInStock - $requestedQty > 0.0001 on such scale?
@orlangur 

Bccomp support negative values:
`$ php -r "echo bccomp((string)+0.0000000500001, (string)-0.0000000000000001, 4);"
0` - equal

` php -r "echo -0.000000000004 - 0 >= 0;"`
`false`